presto-main/src/main/java/com/facebook/presto/operator/aggregation/differentialentropy/Constants.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+public class Constants
+{
+    private Constants() {}
+
+    public static final String RESERVOIR_VASICEK_METHOD_NAME = "reservoir_vasicek";
```

Comment:
these constants are used only in DifferentialEntropyAggregation; hence, move them there and make them package private with @VisibleForTesting if necessary